GDAP Documentation (beta APIs) (#16193)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/tenantrelationship.md

## Relationships
|Relationship|Type|Description|
|:---|:---|:---|

> |delegatedAdminCustomers|[delegatedAdminCustomer](../resources/delegatedadmincustomer.md) collection|The customer who has a delegated admin relationship with a Microsoft partner.|

Reviewer: The customers who have a delegated admin relationship with a Microsoft partner.

> |delegatedAdminCustomers|[delegatedAdminCustomer](../resources/delegatedadmincustomer.md) collection|The customer who has a delegated admin relationship with a Microsoft partner.|

Reviewer: ```suggestion
|delegatedAdminCustomers|[delegatedAdminCustomer](../resources/delegatedadmincustomer.md) collection|The customers who have a delegated admin relationship with a Microsoft partner.|
```

> |delegatedAdminRelationships|[delegatedAdminRelationship](../resources/delegatedadminrelationship.md) collection|The details of the delegated administrative privileges that a Microsoft partner has in a customer tenant.|

Reviewer: ```suggestion
|delegatedAdminRelationships|[delegatedAdminRelationship](../resources/delegatedadminrelationship.md) collection|The list of delegated administrative relationships that a Microsoft partner with multiple customers.|
```
